pkg/pub_integration/lib/src/fake_platform.js

```diff
@@ -0,0 +1,120 @@
+window.gapi = {
+  load: function (name, onReady) {
+    console.log('gapi.load');
+    onReady();
+  },
+  auth2: {
+    init: function (params) {return googleAuth;},
+    getAuthInstance: function () {return googleAuth;},
+  },
+};
+
+function runPubInit() {
+  setTimeout(function () {
+    if ('pubAuthInit' in window) {
+      pubAuthInit();
+    } else {
+      runPubInit();
```

Comment:
Is this hack really necessary? Can't we use `DOMContentLoaded` or something?

Actually, given that it's the last script to be loaded (even with defer), it should also execute after the other scripts..

https://javascript.info/script-async-defer

----
Note: I think we also rely on this behavior in production, so it might be best to test it too :)

Reply:
Added a TODO for it.